components/threatconnect.js

```diff
@@ -253,11 +254,52 @@ polarity.export = PolarityComponent.extend({
 
       Ember.set(caseToUpdate, '__newAttributes', updatedAttributes);
     },
+    cancelRemoveDescription(indicatorId) {
+      const path = `indicators.${indicatorId}.indicator.__newCase`;
+      this.set(`${path}.__selectedWorkflowData`, null);
+    },
+
+    confirmRemoveDescription(indicatorId) {
+      const path = `indicators.${indicatorId}.indicator.__newCase`;
+      const modalData = this.get(`${path}.__selectedWorkflowData`);
+
+      if (modalData) {
+        this.set(`${path}.description`, modalData.templateDescription || '');
+      }
+
+      this.set(`${path}.__selectedWorkflowData`, null);
+    },
     updateNewCaseField(indicatorId, field, event) {
       const value = field === 'associateIndicator' ? event.target.checked : event.target.value;
       const path = `indicators.${indicatorId}.indicator.__newCase`;
 
       this.set(`${path}.${field}`, value);
+
+      if (field === 'workflowTemplate') {
```

Comment:
I didn't see this field being passed in anywhere from the template.  Is it being used?